index.html

```diff
@@ -49,8 +49,8 @@ <h2 class="connect-link">LET’S CONNECT</h2>
         </section>
         <section class="card">
             <article class="card-section">
-                <img src="./images/Snapshoot-Portfolio-4.png" alt="tonic-snapshot" class="tonic-img-mobile">
-                <img src="./images/Snapshoot-Portfolio-4.png" alt="tonic-snapshot" class="tonic-img-desktop">
+                <img src="./images/tonic-img-1.png" alt="tonic-snapshot" class="tonic-img-mobile">
+                <img src="./images/snaposhot-1-desktop-p.png" alt="tonic-snapshot" class="tonic-img-desktop">
                 <article class="card-section-text">
```

Comment:
- [ ] Please fix this spacing issue as mentioned by the previous reviewer, I will drop 2 images below to help you investigate and make the necessary changes
- [ ] [ OPTIONAL ] Your card is not centered, please make use they are aligned in the middle of your page as per the original design, this comment is optional but I would suggest you make it to make your app even better.

###  your design

<img width="1280" alt="1" src="https://user-images.githubusercontent.com/77843050/213794488-34392b28-31ba-407c-a8b3-2e03eda1d79d.png">

### Figma

<img width="491" alt="2" src="https://user-images.githubusercontent.com/77843050/213795069-06251d30-9ddc-4e99-8db2-3e400b4fb250.png">